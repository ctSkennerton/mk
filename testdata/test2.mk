# TEST_MAIN is set by the test harness.
test2.mk.o: $TEST_MAIN
	secondprocess $prereq $stem $target

# TEST_MAIN is mk
mk:
	process bar $target

