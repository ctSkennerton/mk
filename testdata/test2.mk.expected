test2.mk.o: secondprocess mk $stem test2.mk.o
